MyHashMap.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  MyHashMap.swift
+//  coding-swift
+//
+//  Created by Alagu Karthikprabhu on 5/25/20.
+//  Copyright © 2020 kpalagu. All rights reserved.
+//
+
+// Time Complexity : (O)N
+// Space Complexity : (O)N
+// Did this code successfully run on Leetcode : No Timelimit exceeded
```

Comment:
Basically we are creating an array of linked lists. Hope this structure clears it
Try to code it again and lmk if TLE occurs again
[List1_head, List2_head, List3_head]
          |                   |                   |           
List1_node2  List2_node2  List3_node2
          |                   |                   |           
List1_node3  List2_node3  List3_node3